front/app/components/UI/ImagesDropzone/index.tsx

```diff
@@ -214,6 +214,10 @@ class ImagesDropzone extends PureComponent<
   }
 
   componentDidMount() {
+    forEach(this.state.urlObjects, (urlObject) =>
+      window.URL.revokeObjectURL(urlObject)
+    );
+
     this.setUrlObjects();
```

Comment:
We're already setting other state variables in the componentDidMount, so I don't think it's any problem to just move the code into the componentDidMount.

I originally tried to refactor this component into a functional one, but I ran into a lot of issues :grimacing: I'm going to add a ticket to the Platform board instead I think and see if I can revisit it later in the week, but for now I'll move on to the other StrictMode warnings.